docs/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -40,6 +40,14 @@ When the rate limit is triggered, the client will receive an HTTP 429 response.
 
 ## Create a client
 
+Currently, C8 SaaS is supporting the following Scopes:
+
+- Zeebe - Access to the Zeebe gRPC+REST API
```

Comment:
i am actually not sure if we should mention REST here in the first place @christinaausley - any feelings?

Reply:
Looping in @aleksander-dytko and @felix-mueller to see how they want to handle this introduction across the docs.

Reply:
We can mention REST